### ISSUE TITLE
Sync movement

### ISSUE THREAD

**Issue by kartpat** [popabczhang/TangoBattleChess#4]
Networked movement of pieces. Should smoothly update when the other player moves the piece.

**Comment by popabczhang** [comment]
yeah, this is a weird bug. let us know when you figure out why. Thanks!
